src/GenSymIO.chpl

```diff
@@ -644,6 +644,168 @@ module GenSymIO {
         return new MsgTuple(repMsg,MsgType.NORMAL);
     }
 
+    proc readAllParquetMsg(cmd: string, payload: string, st: borrowed SymTab): MsgTuple throws {
+        if !hasParquetSupport {
+            throw getErrorWithContext(
+                   msg="Arkouda has been built without Parquet support",
```

Comment:
2 quick notes:

1. I'm not sure if we should throw an error here or return a MsgTuple with the `MsgType.Error` 
```chapel
return new MsgTuple(msg, MsgType.Error);
```
I know the server loop in `arkouda_sever` is in a try/catch and will handle it, so maybe it's fine.  It seems we have two ways of doing this and I'm not sure which one is "_most correct_" (note: you return a MsgType.Error tuple further below).  I'm curious what others think here.

2. OPTIONAL : I read this message a bit weird because of the passive negative voice (not sure what you call it).  Something like  "Arkouda was not built with Parquet support" might be clearer, your call.

Reply:
Ya, I'd be open to either returning the error message or returning, so whatever anyone has a preference for I think both ways are fine. As Elliot mentioned though, we are aware the error handling is not perfect with this initial PR and were hoping to get this initial work merged before over-emphasizing the error handling (unless that is a problem).